_docs-2/configuration/overview.md

```diff
@@ -59,6 +59,17 @@ They can also be set using {% jlink org.apache.accumulo.core.client.admin.Instan
 client.instanceOperations().setProperty("table.durability", "flush");
 ```
 
+The java api also supports adding, modifying and removing multiple properties in a single operation:
+
+```java
+client.instanceOperations().modifyProperties(properties -> {
+  properties.remove(Property.TABLE_FILE_MAX.getKey());
```

Comment:
Can you set table properties at the instance level?

Reply:
Yes - with set on an instance the become "system" level properties.

Before:

```
root@uno> config -f bloom
-----------+--------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------
SCOPE      | NAME                                 | VALUE
-----------+--------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------
default    | table.bloom.enabled ................ | false
default    | table.bloom.error.rate ............. | 0.5%
default    | table.bloom.hash.type .............. | murmur
default    | table.bloom.key.functor ............ | org.apache.accumulo.core.file.keyfunctor.RowFunctor
default    | table.bloom.load.threshold ......... | 1
default    | table.bloom.size ................... | 1048576
default    | tserver.bloom.load.concurrent.max .. | 4
-----------+--------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------

```

After:

```
jshell> import org.apache.accumulo.core.conf.Property;

jshell> client.instanceOperations().modifyProperties(properties -> {
   ...>   properties.remove(Property.TABLE_FILE_MAX.getKey());
   ...>   properties.put("table.bloom.enabled", "true");
   ...>   properties.put("table.bloom.error.rate", "0.75");
   ...>   properties.put("table.bloom.size", "128000");
   ...> });

root@uno> config -f bloom
-----------+--------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------
SCOPE      | NAME                                 | VALUE
-----------+--------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------
default    | table.bloom.enabled ................ | false
system     |    @override ....................... | true
default    | table.bloom.error.rate ............. | 0.5%
system     |    @override ....................... | 0.75
default    | table.bloom.hash.type .............. | murmur
default    | table.bloom.key.functor ............ | org.apache.accumulo.core.file.keyfunctor.RowFunctor
default    | table.bloom.load.threshold ......... | 1
default    | table.bloom.size ................... | 1048576
system     |    @override ....................... | 128000
default    | tserver.bloom.load.concurrent.max .. | 4
-----------+--------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------

```